lib/core/form/components/widgets/text/text.widget.spec.ts

```diff
@@ -269,10 +279,18 @@ describe('TextWidgetComponent', () => {
                 });
 
                 fixture.detectChanges();
+                inputElement = <HTMLInputElement> element.querySelector('#text-id');
             });
 
             it('should show the input mask placeholder', () => {
-                const inputElement: HTMLInputElement = <HTMLInputElement> element.querySelector('#text-id');
+                expect(inputElement).toBeDefined();
+                expect(inputElement).not.toBeNull();
+                expect(inputElement.placeholder).toBe('Phone : (__) ___-___');
+            });
+
+            it('should show the input mask placeholder when clicked', () => {
```

Comment:
this tests is related to the input mask placeholder that is different from the normal placeholder. Also i don't think the placeholder should be visibile when clicking on the widget.

Reply:
The similar behaviour is in APS also. When we click on the text area the placeholder value is visible. It disappears only when value is entered in the text area.